Always build pacrunners as separate modules, not libproxy builtins
--- libproxy-0.4.18/libproxy/cmake/modules.cmk.orig	2023-02-26 19:37:25.969337262 +0100
+++ libproxy-0.4.18/libproxy/cmake/modules.cmk	2023-02-26 19:51:16.691503520 +0100
@@ -21,12 +21,7 @@ include(cmake/modules/pacrunner_webkit.c
 include(cmake/modules/pacrunner_duktape.cmk)
 
 # Build the pacrunner into libproxy unless we are building for multiple engines
-set(BIPR 1 CACHE BOOL "Build internal pacrunner? (Always false when building more than one PR")
-
-math(EXPR NUM_PACRUNNER "${MOZJS_FOUND}+${NATUS_FOUND}+${WEBKIT_FOUND}+${DUKTAPE_FOUND}")
-if(NUM_PACRUNNER GREATER 1)
-  set(BIPR 0)
-endif()
+set(BIPR 0)
 
 #
 ## Module definition
