--- libproxy-0.4.12/libproxy/cmake/modules/pacrunner_mozjs.cmk.orig	2016-03-02 10:39:39.875263314 +0100
+++ libproxy-0.4.12/libproxy/cmake/modules/pacrunner_mozjs.cmk	2016-03-02 10:43:12.361921064 +0100
@@ -9,7 +9,7 @@
 elseif(NOT APPLE)
   option(WITH_MOZJS "Search for MOZJS package" ON)
   if (WITH_MOZJS)
-    pkg_search_module(MOZJS mozjs185)
+    pkg_search_module(MOZJS mozjs187)
     if(MOZJS_FOUND)
       include_directories(${MOZJS_INCLUDE_DIRS})
       link_directories(${MOZJS_LIBRARY_DIRS})
