--- libproxy-0.4.6/bindings/python/CMakeLists.txt.orig	2010-09-01 22:23:34.000000000 +0200
+++ libproxy-0.4.6/bindings/python/CMakeLists.txt	2010-11-15 21:16:27.219102452 +0100
@@ -18,15 +18,11 @@
       set(libdir "${CMAKE_INSTALL_PREFIX}/${LIB_INSTALL_DIR}")
     endif()
 
-    if (IS_DIRECTORY "${libdir}/python${PYTHON_VERSION}/site-packages")
-      set(PYTHON_SITEPKG_DIR "${libdir}/python${PYTHON_VERSION}/site-packages")
-    else()
       execute_process(COMMAND
 		      ${PYTHON_EXECUTABLE}
 		      -c "import distutils.sysconfig ; print distutils.sysconfig.get_python_lib(plat_specific=0)"
 		      OUTPUT_VARIABLE PYTHON_SITEPKG_DIR
 		      OUTPUT_STRIP_TRAILING_WHITESPACE)
-    endif()
   endif()
   
   message(STATUS "Using PYTHON_SITEPKG_DIR=${PYTHON_SITEPKG_DIR}")
