diff -urN libproxy-0.2.3/configure.ac libproxy-0.2.3.new/configure.ac
--- libproxy-0.2.3/configure.ac	2008-06-05 22:56:16.000000000 +0200
+++ libproxy-0.2.3.new/configure.ac	2009-02-18 23:07:03.000000000 +0100
@@ -180,6 +180,7 @@
 
 ### Checks for library functions.
 AC_FUNC_MALLOC
+AC_SEARCH_LIBS([dlopen], [dl])
 #AC_CHECK_FUNCS([gethostbyname gethostname memset socket strdup strstr])
 
 AC_CONFIG_FILES([libproxy-1.0.pc Makefile src/Makefile src/lib/Makefile
